### PR TITLE
Release Firecracker v1.1.4

### PR DESCRIPTION
**Pull request by pb8o**
## Changes

Fix bug related to ARM64 snapshots

## Reason

...

## License Acceptance

By submitting this pull request, I confirm that my contribution is made under
the terms of the Apache 2.0 license.

## PR Checklist

- [x] All commits in this PR are signed (`git commit -s`).
- [x] If a specific issue led to this PR, this PR closes the issue.
- [x] The description of changes is clear and encompassing.
- [x] Any required documentation changes (code and docs) are included in this PR.
- [x] New `unsafe` code is documented.
- [x] API changes follow the [Runbook for Firecracker API changes][2].
- [x] User-facing changes are mentioned in `CHANGELOG.md`.
- [x] All added/changed functionality is tested.

---

- [ ] This functionality can be added in [`rust-vmm`][1].

[1]: https://github.com/rust-vmm
[2]: https://github.com/firecracker-microvm/firecracker/blob/main/docs/api-change-runbook.md

### CODE REVIEW

**Review by roypat — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## [1.1.4]
+
+### Fixed
+
+- Fixed a bug on ARM64 hosts where the upper 64bits of the V0-V31 FL/SIMD
```
Comment (JonathanWoollett-Light) [inline CHANGELOG.md:7]: Nit space:
```suggestion
- Fixed a bug on ARM64 hosts where the upper 64 bits of the V0-V31 FL/SIMD
```

File: CREDITS.md [hunk 2]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (JonathanWoollett-Light) [inline CREDITS.md:1]: Should this newline be included?

File: CHANGELOG.md (hunk 1 above)
Comment (pb8o) [reply CHANGELOG.md:7]: I copied it from main. It may be worth fixing it there instead.

File: CREDITS.md (hunk 2 above)
Comment (pb8o) [reply CREDITS.md:1]: I agree it shouldn't but I would fix it also in `main`

**Review by JonathanWoollett-Light — APPROVED** [review]

**Pull request merged**